CONTRIBUTING.md

```diff
@@ -166,9 +141,61 @@ On the command line you can use `git commit -s` to sign off the commit.
 * Step 2: make yourself useful by contributing code, bugfixes, support etc.
 * Step 3: volunteer on [the discussion group](https://github.com/openhab/openhab-js/issues?labels=question&page=1&state=open)
 
-Don't forget: being a maintainer is a time investment. Make sure you will have time to make yourself available.
+Don't forget: being a maintainer is a time investment. 
+Make sure you will have time to make yourself available.
 You don't have to be a maintainer to make a difference on the project!
 
+
+## Build system
+
+openhab-js is using NodeJS version 12.16.1 as the JS Scripting add-on is currently on this version.
+It is required to use this version as well in your development environment, otherwise you could cause trouble.
+
+Maintainers recommend to use a NodeJS version manager, see [nvm-sh/nvm](https://github.com/nvm-sh/nvm).
+
+openhab-js has several (development) dependencies which are required.
+
+Run `npm install` to install those dependencies.
+
+A few of those dependencies are required to be installed globally:
```

Comment:
Is this really the case?  I think these are installed as dev dependencies, so no globals are required.  At least for me, i can run everything in package.json without globals.

Reply:
No it is not, thanks for the question.
I just rechecked with a new Node installation, I will correct this part.

I am wondering what made my think that they are required as globals.